bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx60.html

```diff
@@ -0,0 +1,126 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
+
+{% add_lang_files "firefox/whatsnew_59" "firefox/mobile" "firefox/sendto" %}
+
+{% extends "firefox/whatsnew/base-pebbles.html" %}
+
+{% block page_title %}{{ _('See what’s new with Firefox') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_whatsnew_60' %}
+{% endblock %}
+
+{% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
+
+{% block content %}
+<main>
+  {% with custom_message=_('Congrats! You’re using the latest version of Firefox.') %}
+    {% include 'firefox/whatsnew/includes/header.html' %}
+  {% endwith %}
+  <div class="main-content {% if show_send_to_device %}has-send-to-device{% else %}no-send-to-device{% endif %}">
+    <div class="content-container visible" id="logo-container">
+      <div class="logo-wrapper">
+        {{ high_res_img('logos/firefox/logo-quantum.png', {'alt': 'Firefox', 'width': '100', 'height': '100', 'class': 'logo-fx', 'id': 'logo-fx'}) }}
+        {{ high_res_img('logos/firefox/logo-focus.png', {'alt': ('Klar' if LANG == 'de' else 'Focus'), 'width': '100', 'height': '100', 'class': 'logo-focus', 'id': 'logo-focus'}) }}
+      </div>
+    </div>
+    {#
+      If the user is not signed in to Sync, display the FxA iframe.
+    #}
+    <div class="content-container fxa-container" id="fxa-container">
+      <header class="content-header">
+        <h2>{{ _('Take Firefox with You') }}</h2>
+        <p class="tagline">{{ _('Get your bookmarks, history, passwords and other settings on all your devices.') }}</p>
+        <a href="{{ url('firefox.accounts-features') }}" target="_blank" rel="noopener noreferrer">{{_('Learn more about Firefox Accounts')}}</a>
```

Comment:
I understand this `firefox.accounts-features` page is only localized/enabled for en-US? Is it better to use `firefox.accounts` or `firefox.features.sync` here for other locales?

Reply:
Ah, good catch! I will use `firefox.features.sync` for non en-US locales.